src/storekit.cs

```diff
@@ -27,18 +31,22 @@ partial interface SKDownload {
 		[Export ("state")]
 		SKDownloadState State { get; }
 #if MONOMAC
+		[NoWatch] // not necessary b/c #if MONOMAC, so this will not even compile for ios/watch/tvos?
```

Comment:
It seems like this is not necessary because we have `#if MONOMAC` and it was not marked as `[NoTV][NoiOS]` before?

Reply:
no, it's not required
it would tell, in Xamarin.Mac.dll, that this is not available on watchOS - but that's not very useful